source/CardGames.py

```diff
@@ -305,3 +305,7 @@ def showLeaderboard(wins):
     for winner in wins:
       leaderboard += f"{winner.name}: {wins[winner]} wins\n"
     print(leaderboard)
+
+params = initializeGame()
+updateLeaderboard(showWinner(Play(params[0], params[1], params[2])).name)
+showLeaderboard(wins)
```

Comment:
The show leaderboard function doesn't take any parameters

Reply:
Thank you for pointing this out. My version of showLeaderboard is still from the previous sprint which has your old implementation that still takes the global wins dictionary as input. Once we merge for this sprint this will definitely be something that I will need to change per your new implementation of the leaderboard process. Great catch!